src/hw1/Main.java

```diff
@@ -0,0 +1,325 @@
+package hw1;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+// CoworkingSpace: represents a coworking space
+class CoworkingSpace {
+    int spaceID;
+    String type;
+    double price;
+    boolean available;
+
+    public CoworkingSpace(int spaceID, String type, double price, boolean available) {
+        this.spaceID = spaceID;
+        this.type = type;
+        this.price = price;
+        this.available = available;
+    }
+}
+
+// Reservations: represents a reservation
+class Reservations {
+    int reservationID;
+    int coworkingSpaceID;
+    String customerName;
+    String date;
+    String startTime;
+    String endTime;
+
+    public Reservations(int reservationID, int coworkingSpaceID, String customerName, String date, String startTime, String endTime) {
+        this.reservationID = reservationID;
+        this.coworkingSpaceID = coworkingSpaceID;
+        this.customerName = customerName;
+        this.date = date;
+        this.startTime = startTime;
+        this.endTime = endTime;
+    }
+}
+
+public class Main {
+
+    static Scanner scanner = new Scanner(System.in);
+    static ArrayList<CoworkingSpace> spaces = new ArrayList<>(); // list of coworking spaces
+    static ArrayList<Reservations> reservations = new ArrayList<>(); // list of reservations
+    static int reservationIdCounter = 1; // generate a unique reservation ID for each reservation
+
+    public static void main(String[] args) {
+
+        spaces.add(new CoworkingSpace(1, "Open Space", 1500.0, true));
+        spaces.add(new CoworkingSpace(2, "Private Room", 2500.0, true));
+        spaces.add(new CoworkingSpace(3, "Meeting Room", 3000.0, true));
+
+        boolean exit = false;
+
+        while (!exit) {
+            System.out.println("<< Welcome to a Coworking Space Reservation System >>");
```

Comment:
try to use java text blocks here, example:  https://www.baeldung.com/java-text-blocks